### PR TITLE
Fixed error on duplicated includes

### PR DESCRIPTION
**Pull request by lucca65**
We had includes with two methods with the same name. That caused confusion when creating customers. Now fixed. Also bumped version of the gem

### CODE REVIEW

**Comment by vpaciulli** [comment]
+1! gogogogogo

File: lib/kiik/rest/update.rb [hunk 1]
```diff
@@ -19,18 +19,19 @@ def update!(params={})
...
-          response = put url + "/" + params['id'], options
-          response.body
+        def request_update(params)
+          options = opts.merge!(body: JSON.generate(params))
+          response = put(url + "/" + params['id'], options)
           case response.code
           when 200
+            $stdout.puts "BATEU  DO BUILD"
             build(JSON.parse(response.body))
+            $stdout.puts "BATEU DEPOIS DO BUILD"
```
Comment (esdrasedu) [inline lib/kiik/rest/update.rb]: :-1:

**Comment by esdrasedu** [comment]
+1, only remove debug

File: lib/kiik/rest/update.rb (hunk 1 above)
Comment (jonatandahora) [inline lib/kiik/rest/update.rb]: :-1:

**Comment by jonatandahora** [comment]
+1 except BATEU DEPOIS DO BUILD

**Comment by leorog** [comment]
+1

File: lib/kiik/rest/update.rb (hunk 1 above)
Comment (lucca65) [inline lib/kiik/rest/update.rb]: NOOO FORGOT THAT ONE, perdon

**Pull request merged**